### DISCUSSION TITLE
HyperHDR & Vero 4K+

### DISCUSSION THREAD

**Discussion by mkrsn** [awawa-dev/HyperHDR#697] [Q&A]
Hi All,

I currently try to migrate from Hyperion-ng to HyperHDR (20.0.0.0~bullseye~beta1 via official deb repo) but with no luck so far. I'm not using any USB-Grabber - just software grabber. This worked so far very good with Hyperion & Vero 4K+.

In HyperHDR the WebUI only detects /dev/fb0 & /dev/fb1. fb0 containst the Kodi Menü. But as soon as i start a Video then fb0 went black -> no Picture (Video plays on TV) -> no LEDs :(

Unfortunately Hyperion doesn't show which device it is capturing. They just call the Device "amlogic" with input "Display0". From what I could see in the open filedescriptors, hyperion is using /dev/fb0, /dev/amvideo and after starting a video /dev/amvideocap0. The last two are not displayed in HyperHDR.

Does anyone have a Idea what's going on here? Thanks :)

Greetings
Matthias

**Comment by awawa-dev** [comment]
Hi
Currently, the presence of an FB device is tested using a very simple file pattern:
https://github.com/awawa-dev/HyperHDR/blob/558c7e03d7da0d52653cd2fe68d96e3634fae227/sources/grabber/framebuffer/FrameBufGrabber.cpp#L192
But I'm not sure if adding an Amlogic specific FB path would solve your problem or if something else needs to be implemented also.

**Comment by mkrsn** [comment]
A quick look into the [Hyperion Sources](https://github.com/hyperion-project/hyperion.ng/tree/master/src/hyperion-aml) shows that there is a bit more necessary. It's not that much, but sadly more than just adding /dev/amvideo* to the File pattern.
